locations/spiders/farmershomefurniture.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+import re
+
+import scrapy
+from locations.hours import OpeningHours
+from locations.items import GeojsonPointItem
+
+DAYS = {
+    "0": "Mo",
+    "1": "Tu",
+    "2": "We",
+    "3": "Th",
+    "4": "Fr",
+    "5": "Sa",
+    "6": "Su"
+}
+
+
+class FarmersHomeFurnitureSpider(scrapy.Spider):
+    name = 'farmershomefurniture'
+    item_attributes = {'brand': 'Farmers Home Furniture'}
+    allowed_domains = ['www.farmershomefurniture.com']
+    start_urls = ['https://www.farmershomefurniture.com/store-list.inc']
+    download_delay = 1
+    headers = {
+        "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_1) AppleWebKit/537.36 (KHTML, like Gecko) "
+                  "Chrome/63.0.3239.84 Safari/537.36"
+    }
+
+    def parse(self, response):
+        for store in response.xpath('//tr'):
+            store_data = store.xpath('./td/text()').getall()
+            if store_data:
+                properties={ ['city', 'state', 'address', 'phone'][i]: store_data[i]
+                                                        for i in range(len(store_data)) }
```

Comment:
This for loop is hard to read and seems to be unnecessary. Can you split the four properties into their own lines?